### ISSUE TITLE
Include with relative path does not work

### ISSUE THREAD

**Issue by windsource** [runceel/plantumlpreview#6]
I use version 0.0.12 and in this version including a skin via a relative path does not work anymore.

Example:

```
@startuml

!include ../clean.skin

class Person {
}

@enduml
```

The error message I get is:

`Cannot include C:\Users\username\AppData\Local\Temp\okazukiplantuml\..\clean.skin`

**Comment by runceel** [comment]
ah...

It is very difficult problem. Realtime preview step to, copy file to temp, execute command to temp folder file. Temp folder file dosen't have relative files.

I 'll try to think of a solution .

**Comment by runceel** [comment]
I released v0.0.13.

Limited support `include with relative path`.
- Open all included file and preview.
- Open and edit file.

**Comment by windsource** [comment]
Thanks. Now it works!
